examples/abi/Cargo.lock

```diff
@@ -1239,3 +1266,17 @@ dependencies = [
  "syn",
  "synstructure",
 ]
+
+[[package]]
+name = "zeropool-bn"
+version = "0.5.11"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "71e61de68ede9ffdd69c01664f65a178c5188b73f78faa21f0936016a888ff7c"
+dependencies = [
+ "borsh",
+ "byteorder",
+ "crunchy",
+ "lazy_static",
+ "rand 0.8.5",
+ "rustc-hex",
+]
```

Comment:
why is it that these other deps are being pulled in?

Reply:
`near-vm-logic`'s dependency on `zeropool-bn` used to be optional. It no longer is.

Reply:
the `near-vm-logic` dep didn't change with this, did it? Was that an accidental update? Just trying to make sure the side effects are clear

Reply:
Yeah, I don't understand why it just updated wither. Looking at the lockfile, `near-vm-logic` didn't change. Then again, it's limited to just the `abi` example. So any side effects should be sandboxed in there.

Reply:
I think we forgot to update the lockfile in this PR: https://github.com/near/near-sdk-rs/pull/885. Ever since we got rid of the examples diff check we are not enforcing lockfile to be up to date.

Reply:
ah, yes, we should probably add the git diff check back in